modules/Login/pages/Login.tsx

```diff
@@ -0,0 +1,41 @@
+import Button from '@modules/shared/components/atoms/Button/Button';
+import type { FC, ReactElement } from 'react';
+import { useEffect } from 'react';
+import { useRouter } from 'next/router';
+import * as EButton from '../../shared/components/atoms/Button/types/EButton';
+import { loginUser, register } from '../../shared/api/auth';
+
+const Login: FC = (): ReactElement => {
+  const router = useRouter();
+  useEffect(() => {
+    if (document.cookie.split('=')[0] === 'user') {
```

Comment:
this logic "determining whether the user is logged in or not is repeated. we discussed that this should be managed by a custom hook for cleaner code (I am aware of Elhaw suggestion for later improvement but this still not good enough to be merged)